src/btree/bt_read.c

```diff
@@ -247,11 +247,23 @@ __wt_page_in_func(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags
                 return (WT_NOTFOUND);
             goto read;
         case WT_REF_DISK:
-            /* Limit reads to cache-only, or internal pages only. */
-            if (LF_ISSET(WT_READ_CACHE) ||
-              (LF_ISSET(WT_READ_CACHE_LEAF) && __wt_ref_is_leaf(session, ref)))
+            /* Limit reads to cache-only */
```

Comment:
Minor nit: please end comment with a period at the end of the sentence.

Reply:
Fixed.